cmd/harmony/main.go

```diff
@@ -322,9 +326,15 @@ func setupConsensusAndNode(nodeConfig *nodeconfig.ConfigType) *node.Node {
 		currentConsensus.DisableViewChangeForTestingOnly()
 	}
 
+	blacklist, err := setupBlacklist()
+	if err != nil {
+		_, _ = fmt.Fprintf(os.Stderr, "Blacklist error: %s\n", err.Error())
```

Comment:
If there is no blacklist file, we should just continue normally. we shouldn't block the node from running by blacklist.

Reply:
Yes, that's a good idea, will be updated on latest